session/keepalive/example_test.go

```diff
@@ -23,14 +24,21 @@ var (
 	keepAliveIdle      = sessionIdleTimeout / 2
 )
 
-func init() {
-	simulator.SessionIdleTimeout = sessionIdleTimeout
-}
-
 func ExampleHandlerSOAP() {
-	simulator.Run(func(ctx context.Context, c *vim25.Client) error {
-		// No need for initial Login() here as simulator.Run already has
-		m := session.NewManager(c)
+	simulator.Run(func(ctx context.Context, vc *vim25.Client) error {
+		// Using the authenticated vc client, timeout will apply to new sessions.
+		session.NewManager(vc).SetSessionTimeout(ctx, sessionIdleTimeout.String())
```

Comment:
IIUC `sessionIdleTimeout` is `250ms` and will be represented that way when converted to `string`, yet the comment for `SetSessionTimeout` says that it expects the value in minutes? Based on that comment for `SetSessionTimeout` I'd expect the value passed in here to be `1` or `0.001`.